BREAKING.md

```diff
@@ -41,6 +42,9 @@ Use IContainer.close() or IContainerContext.closeFn() instead, and pass an error
 Offline load functionality has been placed behind a feature flag as part of [ongoing offline work](https://github.com/microsoft/FluidFramework/pull/9557).
 In order to use `IContainer.closeAndGetPendingLocalState`, pass a set of options to the container runtime including `{ enableOfflineLoad: true }`.
 
+### connect() and disconnect() made mandatory on IContainer and IFluidContainer
+The functions `IContainer.connect()`, `IContainer.disconnect()`, `IFluidContainer.connect()`, and `IFluidContainer.disconnect()` have all been changed from optional to mandatory functions.
```

Comment:
Do we expect external implementations of these interfaces? Does Bohemia have their own?

Reply:
As far as I can tell they just import our interfaces, so we should be good